src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -719,6 +722,13 @@ event::ptr primitive_inst::execute(const std::vector<event::ptr>& events) {
             }
         }
 
+        if (has_inner_body()) {
```

Comment:
Can't we do this in update_shape?
Like:
`in update_shape() {
   if _node.is_type<condition>()
       execute inner netowkr 
       wait_for_completion of inner networks 
      ...

}`

Reply:
Override execute() function for condition_inst.